.github/workflows/danger.yml

```
@@ -5,17 +5,18 @@ on: [pull_request]
 jobs:
   build:
     name: Danger
-    runs-on: ubuntu-latest
+    runs-on: macos-14
 
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout
+        uses: actions/checkout@v1
```

I doubt we want to downgrade this?
```suggestion
        uses: actions/checkout@v3
```

---

Yup, sorry, copy pasted the wrong thing